packages/ui/src/components/collectibles/native/collectible-card.native.tsx

```diff
@@ -0,0 +1,31 @@
+import { CollectibleHtml } from './collectible-html.native';
+import { CollectibleImage } from './collectible-image.native';
+import { CollectibleText } from './collectible-text.native';
+
+export interface CollectibleCardProps {
+  name: string;
+  type: 'inscription' | 'stacks';
+  src: string;
+  mimeType?: string | null;
+}
+
+export function CollectibleCard({ name, type, src, mimeType }: CollectibleCardProps) {
+  const isOrdinal = type === 'inscription';
+
+  if (isOrdinal) {
+    switch (mimeType) {
+      // TODO: add audio support
```

Comment:
I didn't implement `mimeType === 'audio'` yet. It seems like on web we just show a Head phone icon unless I am missing something

Reply:
Yep, that is correct. We didn't want the user to interact with the controls in the iframe so we just render an icon with a link to the audio.

Reply:
OK thanks. I updated to add basic support just showing an icon so something will appear